src/features/hotspots/details/HotspotDetails.tsx

```diff
@@ -28,17 +33,21 @@ import { getSyncStatus, isRelay } from '../../../utils/hotspotUtils'
 import ShareHotspot from '../../../components/ShareHotspot'
 import TouchableOpacityBox from '../../../components/TouchableOpacityBox'
 import Articles from '../../../constants/articles'
+import HotspotListItem from '../../../components/HotspotListItem'
+import { distance } from '../../../utils/location'
```

Comment:
I didn't realize we implemented the haversine formula in the app. I was using a library for disco to do it, but that bit got removed. `"haversine-distance": "^1.2.1"`. We should probably use it or remove it.

Reply:
Ill just go with out local one, its a pretty simple function